<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMain</class>
 <widget class="QWidget" name="WidgetMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Servidor TCP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Direcciones IP disponibles</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="cbxIPs"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnEncenderServer">
       <property name="text">
        <string>Encender Servidor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelServerStatus">
     <property name="text">
      <string>Servidor Apagado</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Mensajes recibidos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEditChat"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Mensaje a enviar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Escriba su mensaje</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnEnviarChat">
     <property name="text">
      <string>Enviar Chat</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
